IOS/SwiftSeedProject/SwiftSeedProject/Infrastructure/InfoUtils.swift

```diff
@@ -0,0 +1,20 @@
+//
+//  InfoUtils.swift
+//  SwiftSeedProject
+//
+//  Created by Brian Sztamfater on 21/4/17.
+//  Copyright © 2017 Making Sense. All rights reserved.
+//
+
+import Foundation
+
+class InfoUtils {
+    
+    static func bundleNameKey() -> String {
+        guard let info = Bundle.main.infoDictionary,
```

Comment:
Watch out for the alignment here, it should look like this:

```Swift
guard let info = Bundle.main.infoDictionary,
      let domain = info[kCFBundleNameKey as String] as? String else {
    fatalError("Cannot get bundle name key from Info.plist")
}
```

(Golden Path section on our Style-guide)